docs/sdk/web/providers/evm.mdx

```diff
@@ -47,6 +47,11 @@ await web3auth.initModal();
 
 web3auth.connect();
 ```
+## Blockchain Providers
+
+- [Alchemy](https://www.alchemy.com/). As a developer platform, Alchemy provides a suite of developer toolings and abstractions including JSON RPC APIs across all major chains, an ethers.js SDK, and a library of [enhanced APIs](https://docs.alchemy.com/reference/enhanced-apis-overview) like their NFT APIs. 
+
```

Comment:
we should add other providers like infura, quicknode and ankr here.

Reply:
Sg, just changed it :) Didn't want to misspeak on their details